Tests/winswTests/MainTest.cs

```diff
@@ -0,0 +1,45 @@
+﻿using NUnit.Framework;
+using winsw;
+using winswTests.Util;
+
+namespace winswTests
+{
+    [TestFixture]
+    class MainTest
+    {
+
+        [Test]
+        public void PrintVersion()
+        {
+            string expectedVersion = WrapperService.Version.ToString();
+            string cliOut = CLITestHelper.CLITest(new[] { "version" });
+            StringAssert.Contains(expectedVersion, cliOut, "Expected that version contains " + expectedVersion);
+        }
+
+        [Test]
+        public void PrintHelp()
+        {
+            string expectedVersion = WrapperService.Version.ToString();
+            string cliOut = CLITestHelper.CLITest(new[] { "help" });
+            
+            StringAssert.Contains(expectedVersion, cliOut, "Expected that help contains " + expectedVersion);
+            StringAssert.Contains("start", cliOut, "Expected that help refers start command");
+            StringAssert.Contains("start", cliOut, "Expected that help refers help command");
```

Comment:
Might want to change this test to validate based on your error message.  This one is looking for start but error is for help.  Assume copy/paste issue.